src/js/languages/default.json

```diff
@@ -19,6 +19,10 @@
     "enterCalendar": "Calendar is open, use arrow keys and enter to select a date.",
     "exitCalendar": "Exited calendar dialog"
   },
+  "dataChart": {
+    "detailTitle": "Data chart",
+    "detailFocus": "Use the left and right arrow keys to navigate the chart data."
+  },
```

Comment:
These need to be added to the messages property definition in Grommet/index.d.ts and propTypes.js.

Reply:
Done!